pulsar_spectra/catalogue_papers/Sieber_1973_raw_to_yaml.py

```diff
@@ -46,8 +46,9 @@
             fact_err = 0.5
         elif "c" in row[4]:
             fact_err = 0.75
-        pulsar_dict[pulsar]["Flux Density error mJy"] += [round(flux*fact_err, 3)]
+
+        pulsar_dict[pulsar]["Flux Density error mJy"] += [round(flux*fact_err/query['P0'][pid], 3)]
 
 with open("Sieber_1973.yaml", "w") as cat_file:
-    cat_file.write(json.dumps(pulsar_dict))
+    cat_file.write(json.dumps(pulsar_dict, indent=4))
```

Comment:
It's good that you've remembered to do this but I think we should use indent=1. Just so the file is smaller. We should eventually do this for all of the yamls.